build/stf-run-ci/tasks/main.yml

```diff
@@ -13,11 +15,22 @@
     sg_bridge_image_path: "{{ __internal_registry_path }}/{{ namespace }}/sg-bridge:{{ sg_bridge_image_tag }}"
     prometheus_webhook_snmp_image_path: "{{ __internal_registry_path }}/{{ namespace }}/prometheus-webhook-snmp:{{ prometheus_webhook_snmp_image_tag }}"
 
+- name: Set default image paths for bundle and index builds
+  set_fact:
+    sgo_bundle_image_path: "{{ __internal_registry_path }}/{{ namespace }}/smart-gateway-operator-bundle:{{ sgo_bundle_image_tag }}"
+    sto_bundle_image_path: "{{ __internal_registry_path }}/{{ namespace }}/service-telemetry-operator-bundle:{{ sto_bundle_image_tag }}"
+    stf_index_image_path: "{{ __internal_registry_path }}/{{ namespace }}/service-telemetry-framework-index:{{ stf_index_image_tag }}"
+
 - name: Fail on mutually exclusive flags
   fail:
     msg: __deploy_from_bundles_enabled not currently supported with __local_build_enabled (but should be)
   when: __local_build_enabled | bool and __deploy_from_bundles_enabled | bool
 
+- name: Fail when deploying from index image and local build disabled
+  fail:
+    msg: __deploy_from_index_enabled must also have __local_build_enabled
+  when: __deploy_from_index_enabled | bool and not __local_build_enabled | bool
+
```

Comment:
I do still think it would be useful to add because your second example fails with `__deploy_from_bundles_enabled not currently supported with __local_build_enabled (but should be)`  and we are discussing above that this limitation (and therefor this failsafe) is potentially on the verge of being removed.

Reply:
That was kind of my instinct as well. It certainly doesn't hurt to have another catch in case earlier checks works eventually.

Reply:
This was added. Resolving thread.